### PR TITLE
Harmonize .gitignore

### PR DESCRIPTION
**Pull request by hannesa2**
To avoid redundant entries

### CODE REVIEW

File: android/.gitignore [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
-local.properties
-.idea
 build/
 .DS_Store
 *.iml
 *.apk
 *.aar
 *.zip
-google-services.json
+googleservices.json
```
Comment (calren) [inline android/.gitignore:8]: should this be `google-services`?

File: android/.gitignore [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
 
 .project
 .settings
 .classpath
+
+local.properties
+
+.idea/*
+!idea/codeStyles
```
Comment (calren) [inline android/.gitignore:17]: hm, is there a reason we need this now?

File: android/.gitignore (hunk 1 above)
Comment (hannesa2) [reply android/.gitignore]: You are right. I fixed it

File: android/.gitignore (hunk 2 above)
Comment (hannesa2) [reply android/.gitignore:17]: This helps you to have same codestyle for all devs

File: android/.gitignore [hunk 3]
```diff
@@ -1,6 +1,4 @@
 .gradle
-local.properties
-.idea
```
Comment (zhouyiself) [inline android/.gitignore:3]: Those are still useful when importing material-showcase as a single project.

Comment (zhouyiself) [reply android/.gitignore:3]: Never mind, they're included in parent's .gitignore file.

**Pull request merged**